### ISSUE TITLE
Ensure that StringParseXML supports APIs that have no "xmlns" attribute

### ISSUE THREAD

**Issue by tmpolaczyk** [witnet/witnet-rust#2154]
Check that the StringParseXML operator supports the format returned by this API:

https://forecast.weather.gov/MapClick.php?lat=39.75&lon=-104.99&unit=0&lg=english&FcstType=dwml

**Comment by tmpolaczyk** [comment]
Actually, this example does have two different `xmlns` attributes: `xmlns:xsd` and `xmlns:xsi`:

```xml
<dwml version="1.0" xmlns:xsd="https://www.w3.org/2001/XMLSchema" xmlns:xsi="https://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://graphical.weather.gov/xml/DWMLgen/schema/DWML.xsd">
```

**Comment by tmpolaczyk** [comment]
Fixed by #2156
